src/views/certifications/views/landing/components/certification-program-summary-card/certification-program-summary-card.module.css

```diff
@@ -0,0 +1,31 @@
+.splitSection {
+	padding: 8px 8px 0 8px;
+
+	@media (--dev-dot-tablet-up) {
+		padding: 24px 24px 0 24px;
```

Comment:
```suggestion
		padding: 24px 0 0 24px;
```
I'm not sure if this was design's intent but in figma the right side of this section doesn't have padding 💭 Might be good to verify? 
<img width="607" alt="Screen Shot 2022-12-14 at 3 06 54 PM" src="https://user-images.githubusercontent.com/36613477/207734944-72317eed-aea8-4399-bc61-1ff9b616b344.png">
<img width="789" alt="Screen Shot 2022-12-14 at 3 06 58 PM" src="https://user-images.githubusercontent.com/36613477/207734948-09c14ab5-d73d-4927-a947-175b040d927b.png">

Reply:
Ah great eye - I'll verify and address this in VQA 👍